src/task/src/wrappers/octokitWrapper.ts

```diff
@@ -1,19 +1,23 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
 
-import { OctokitOptions } from '@octokit/core/dist-types/types'
-import { Octokit } from 'octokit'
-import { singleton } from 'tsyringe'
-import OctokitGitDiffParser from '../git/octokitGitDiffParser'
 import CreateIssueCommentResponse from './octokitInterfaces/createIssueCommentResponse'
 import CreateReviewCommentResponse from './octokitInterfaces/createReviewCommentResponse'
 import DeleteReviewCommentResponse from './octokitInterfaces/deleteReviewCommentResponse'
 import GetIssueCommentsResponse from './octokitInterfaces/getIssueCommentsResponse'
 import GetPullResponse from './octokitInterfaces/getPullResponse'
 import GetReviewCommentsResponse from './octokitInterfaces/getReviewCommentsResponse'
 import ListCommitsResponse from './octokitInterfaces/listCommitsResponse'
+import { Octokit } from 'octokit'
+import OctokitGitDiffParser from '../git/octokitGitDiffParser'
+import { OctokitOptions } from '@octokit/core/dist-types/types'
 import UpdateIssueCommentResponse from './octokitInterfaces/updateIssueCommentResponse'
 import UpdatePullResponse from './octokitInterfaces/updatePullResponse'
+import { singleton } from 'tsyringe'
+
+/* eslint-disable camelcase -- Required for alignment with Octokit. */
```

Comment:
does this just disable the check for this file or form this file on?

Reply:
It's just for the file, but to make the behaviour more explicit and to avoid the risk of it changing in the future, I've added explicit disables to the end of each file now.